pandas/core/apply.py

```diff
@@ -554,7 +561,20 @@ def apply_multiple(self) -> DataFrame | Series:
         result: Series, DataFrame, or None
             Result when self.f is a list-like or dict-like, None otherwise.
         """
-        return self.obj.aggregate(self.f, self.axis, *self.args, **self.kwargs)
+        if self.axis == 1 and isinstance(self.obj, ABCDataFrame):
+            return self.obj.T.apply(self.f, 0, *self.args, **self.kwargs).T
```

Comment:
Needs to be `args=self.args`, not `*self.args`. Can you add a test here?

Reply:
Ye, good catch. Done.